compiler/rustc_session/src/config.rs

```diff
@@ -2135,11 +2116,6 @@ fn check_error_format_stability(
     }
     let format = match format {
         ErrorOutputType::Json { pretty: true, .. } => "pretty-json",
-        ErrorOutputType::HumanReadable { kind, .. } => match kind {
-            HumanReadableErrorType::AnnotateSnippet { unicode: false, .. } => "human-annotate-rs",
-            HumanReadableErrorType::AnnotateSnippet { unicode: true, .. } => "human-unicode",
```

Comment:
This shouldn't remove support for `human-unicode` as is [being tracked separately](https://github.com/rust-lang/rust/issues/148607) from using `annotate-snippets` as the default emitter.